tests/src/kinAccount.unit.test.ts

```diff
@@ -243,18 +245,21 @@ describe("KinAccount.createAccount", async () => {
 
 function mock400SendKinResponse(response: ErrorResponse) {
 	nock(fakeUrl)
+		.matchHeader(headerKey, headerVal)
```

Comment:
👍 now we're testing the headers.
In unit test, I would do a dedicate test just for headers, this is helpful for both isolation (adds noise to other tests) and locality (pinpoint the issue)